src/app/_components/how-to-play.tsx

```diff
@@ -54,19 +53,18 @@ export const HowToPlay: FunctionComponent = () => {
             <Typography level="h3">Nominations</Typography>
 
             <Typography>
-              Search and nominate builders on Farcaster or Talent Passport.
-              Nominations reset every Tuesday and your weekly budget gets
-              re-calculated.
+              Cast "nominate" and username in /build on Farcaster to gift $BUILD
+              tokens. Casts liked by curators will share the weekly reward pool.
             </Typography>
 
             <Button
-              href={"/stats"}
+              href="https://rounds.wtf/build"
               component={Link}
               variant="solid"
               color="neutral"
               sx={{ mt: 2 }}
             >
```

Comment:
Here you probably want to open this link on a new tab when a user clicks on it. you can do this by adding a `target="_blank"` property.

```jsx
<Button
  href="https://rounds.wtf/build"
  target="_blank"
  component={Link}
  variant="solid"
  color="neutral"
  sx={{ mt: 2 }}
>
```